library/src/main/java/in/krharsh17/barview/BarGroup.java

```diff
@@ -437,23 +433,26 @@ int dp(float dp) {
      */
     public Drawable getRoundRect(String color,int currentLayer,int totalLayer,int radius) {
 
+        int localRadius=radius;
```

Comment:
Why this?

Reply:
This was an issue @krharsh17  that we should not assign passes in param values rather we should create local variables for assigning tasks and inner manipulation.

Reply:
Oh, okay. And are you sure that this won't break any functionality?

Reply:
no I guess it wont you can see the changes below